### ISSUE TITLE
Document security issues

### ISSUE THREAD

**Issue by bastik-1001** [sandboxie-plus/Sandboxie#3473]
### Is your feature request related to a problem or use case?

There are SECURITY ISSUE IDs, but there is no way to learn what was the deeper issue behind it nor how much of an impact it could have had.

Additionally, those could be updated, before and after some release contained a fix, if there is a workaround, like using a different box-type or changing some setting.

### Describe the solution you'd like

Document security issues with an explanation of what went wrong, what could have been the impact, and if it could have been mitigated by the user.

**Comment by DavidXanatos** [comment]
I have added the informations from the changelog to security.md

**Comment by bastik-1001** [comment]
Yes, this is what I meant. I was not sure, if you wanted to use that file. Now, users can easily see how security has approved over time.
